backend/src/main/java/ai/verta/modeldb/versioning/blob/diff/DiffComputer.java

```diff
@@ -117,6 +117,8 @@ public static AutogenBlobDiff computeBlobDiff(AutogenBlob a, AutogenBlob b) {
             .setEnvironment(
                 computeDiff(
                     a, b, AutogenBlob::getEnvironment, DiffComputer::computeEnvironmentDiff)));
+    // .setDescription(computeDiff(a, b, AutogenBlob::getDescription,
+    // DiffComputer::computeDescriptionDiff)));
```

Comment:
this was commented after the file was auto generated?
Could this be resolved in the templating logic?
cc @conradoverta  if he has any thoughts.